samples/samples-js/host.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "excludedTypes": "Request"
+      }
+    }
+  },
+  "extensionBundle": {
+    "id": "Microsoft.Azure.Functions.ExtensionBundle.Preview",
+    "version": "[3.*, 4.0.0)"
```

Comment:
Might need to bump this to 4.* so we get the casing fix

Although we need to figure out what to do here since right now this means that this is going to be running against the bundle version of the extension and NOT the current code so that kind of defeats the purpose of having tests.

Reply:
yeah while running tests locally, am removing this from host.json. Not sure how to suppress this on build machine, any ideas?